apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/CoinControlModal/components/Row/NoOrdinalsRow.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Flex, Text } from "@ledgerhq/react-ui";
+import { SplitAddress, Cell } from "~/renderer/components/OperationsList/AddressCell";
+import { useTranslation } from "react-i18next";
+
+type Props = {
+  outputIndex: number;
+  hash: string;
+  address: string;
+};
+
+export const NoOrdinalsRow: React.FC<Props> = ({ outputIndex, hash, address }) => {
```

Comment:
while this file represent the `NoOrdinalsRow` main component, it's better to export it as a default.